### PR TITLE
Fixing locating first-use variables from disassembly

### PR DESCRIPTION
**Pull request by mahaloz**
* changed what is stored in the insmap 
* fixed old highlighting of variables by re-adding CVariables to posmap

### CODE REVIEW

File: angr/analyses/decompiler/structured_codegen.py [hunk 1]
```diff
@@ -177,21 +177,21 @@ def mapper(chunks, posmap, stmt_posmap, insmap):
                     elif isinstance(obj, CExpression) and hasattr(obj, 'tags') and \
                             obj.tags is not None and 'ins_addr' in obj.tags:
 
-                        # assure that variables are only placed once in the GUI at the defenition of each var
+                        # assure that variables are only placed once in the GUI at the defn of each var
```
Comment (ltfish) [inline angr/analyses/decompiler/structured_codegen.py:180]: Let's spell out these words when writing comments. There is no need to abbreviate.

File: angr/analyses/decompiler/structured_codegen.py [hunk 2]
```diff
@@ -404,17 +404,17 @@ def c_repr_chunks(self, indent=0):
 
         indent_str = self.indent_str(indent=indent)
 
-        yield indent_str, self
+        yield indent_str, None
```
Comment (ltfish) [inline angr/analyses/decompiler/structured_codegen.py:407]: I feel I've seen these changes before... did you rebase on master?

**Review by ltfish — CHANGES_REQUESTED** [review]

File: angr/analyses/decompiler/structured_codegen.py (hunk 2 above)
Comment (mahaloz) [reply angr/analyses/decompiler/structured_codegen.py]: just rebased

File: angr/analyses/decompiler/structured_codegen.py (hunk 1 above)
Comment (mahaloz) [reply angr/analyses/decompiler/structured_codegen.py]: kk

**Review by ltfish — APPROVED** [review]

**Pull request merged**